tools/deb/istio-start.sh

```diff
@@ -86,6 +88,10 @@ if [ -z "${PILOT_ADDRESS:-}" ]; then
   PILOT_ADDRESS=istio-pilot.${ISTIO_SYSTEM_NAMESPACE}:${ISTIO_PILOT_PORT}
 fi
 
+if [ -z "${ZIPKIN_ADDRESS:-}" ]; then
```

Comment:
I don't think we want to default to zipkin enabled - and long term we don't want to run zipkin in istio-system ( security reasons). 

Please keep it off by default, user should configure ZIPKIN_ADDRESS if he wants traces.